network/interface.go

```diff
@@ -29,9 +29,9 @@ const MaxReprocessBufferSize = 1000000
 
 // Transactions is the interface that defines the API for creating, reading and subscribing to Nuts Network transactions.
 type Transactions interface {
-	// Subscribe makes a subscription for the specified transaction type. The receiver is called when a transaction
-	// is received for the specified event and payload type.
-	Subscribe(eventType EventType, payloadType string, receiver Receiver)
+	// Subscribe registers a receiver for the specified transaction type. The receiver is called when a transaction
+	// matches with the given dag.NotificationFilter's.
```

Comment:
`dag.NotificationFilter` is not in signature? (so, not given)

Reply:
changed comments